src/alfasim_sdk/testing/fixtures.py

```diff
@@ -162,6 +163,21 @@ def run(self, *, number_of_threads: int = -1) -> Results:
                 ],
                 cwd=project_folder,
             )
+        except CalledProcessError as error:
+            results = Results(self.alfacase_data_folder)
+            log_calc = results.log_calc
+            log = results.log
+            msg = textwrap.dedent(
+                f"""\
+                ========================================
+                === log_calc.txt
+                {log_calc.read_text() if log_calc.is_file() else "<not found>"}
+                ========================================
+                === log.txt
+                {log.read_text() if log.is_file() else "<not found>"}
+                """
+            )
+            raise RuntimeError(msg) from error
```

Comment:
Don't need to change.
I just update something very similar to this in alfasim.
It will be something like:

```suggestion
            def list_log(log_file):
                try:
                    contents = log_file.read_text()
                except Exception as e:
                    contents = f'<ERROR READING FILE: {str(e)}>'

                name = log_file.name
                header_len = 4 + len(name) + 4
                return f'{"=" * header_len}\n=== {log_file.name} ===\n{"=" * header_len}\n{contents}'
                
            results = Results(self.alfacase_data_folder)
            log_calc = results.log_calc
            log = results.log
            msg = '\n'.join([list_log(log_calc), list_log(log)])
            raise RuntimeError(msg) from error
```